config-connector/solutions/iam/kpt/service-account/Kptfile

```diff
@@ -0,0 +1,29 @@
+apiVersion: kpt.dev/v1alpha1
+kind: Kptfile
+metadata:
+  name: .
+packageMetadata:
+  shortDescription: sample description
```

Comment:
should be a short description of package

Reply:
Added. Thanks!